tests/integration/modules/mac_user.py

```diff
@@ -94,9 +92,26 @@ def test_mac_user_delete(self, grains=None):
         except CommandExecutionError:
             raise
 
-    @destructiveTest
```

Comment:
We would we remove this decorator on a test that's removing a user from a system?